src/components/schedule-page/toolbar/toolbar-editing.component.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Button } from "../../common-components";
+import { Box } from "@material-ui/core";
+import ValidationDrawerComponent from "../validation-drawer/validation-drawer.component";
+import backend from "../../../api/backend";
+import { ScheduleErrorActionType } from "../../../state/reducers/schedule-errors.reducer";
+import { ActionModel } from "../../../state/models/action.model";
+import { ScheduleError } from "../../../common-models/schedule-error.model";
+import { useDispatch, useSelector } from "react-redux";
+import { ApplicationStateModel } from "../../../state/models/application-state.model";
+import SchedulePage from "../schedule-page.component";
+
+export function ToolbarEditingComponent(props): JSX.Element {
```

Comment:
Przydałby się typ propsów

Reply:
> Przydałby się typ propsów

Czyli słownik? Jeszcze nie do końca rozumiem jak to ma wyglądać

Reply:
Widzę, że propsy już nie są używane, więc `resolve`